.github/workflows/update-and-publish-helm-chart.yml

```diff
@@ -29,14 +29,15 @@ jobs:
       - name: Publish the new helm chart and update index.yaml
         run: |
           helm package cromwell-helm/
-          mv cromwell-${NEXT_VERSION}.tgz charts/cromwell-${NEXT_VERSION}.tgz
+          mv cromwell-${NEXT_VERSION}.tgz charts
```

Comment:
TOL: This looks like it wouldn't have made any different to me since the filename isn't changed?

Reply:
Right. Do you want me to revert it?

Reply:
(Not Chris...) I have a mild preference for being as clear as possible, which I guess favors including the destination filename

Reply:
Sounds fair enough to me. I will revert this change and test it out